app/Http/Resources/ArtistResource.php

```diff
@@ -18,7 +18,7 @@ public function toArray($request)
             'id' => $this->id,
             'label' => new LabelResource($this->label),
             'profile' => new ProfileResource($this->profile),
-            'catalog_profile' => new CatalogResource($this->whenLoaded('catalogable')),
+            'catalog_profile' => new CatalogResource($this->whenLoaded('catalogable.profile')),
```

Comment:
Ok, but the CatalogResource has exclusively calls to profile like this:

```php
        return [
            'id' => $this->id,
            'moniker' => $this->profile->moniker,
            'alt_moniker' => $this->profile->alt_moniker,
            'city' => $this->profile->city,
```

So how should all of this be updated?

Reply:
Actually, let's leave this as you have it, for now. It doesn't break any tests, and I think it could evolve into a longer conversation that wanders way outside the scope of this PR.